### PR TITLE
Retain isolation scheme in Import Peptide Search

### PR DESCRIPTION
**Pull request by kaipot**

### CODE REVIEW

File: pwiz_tools/Skyline/TestTutorial/DiaTutorialTest.cs [hunk 1]
```diff
@@ -327,7 +327,7 @@ protected override void DoTest()
             RestoreViewOnScreen(27);
             PauseForScreenShot<GraphChromatogram>("Chromatogram graph metafile", 26);
 
-            RunUI(SkylineWindow.SelectedNode.Expand);
+            RunUI(() => SkylineWindow.SelectedNode.Expand());
```
Comment (brendanx67) [inline pwiz_tools/Skyline/TestTutorial/DiaTutorialTest.cs:330]: This change seems irrelevant and is not changing what the test does. Probably should not be included.

**Review by brendanx67 — APPROVED** [review]
Looks good, as long as the DiaTutorialTest change is reverted.

File: pwiz_tools/Skyline/SettingsUI/FullScanSettingsControl.cs [hunk 2]
```diff
@@ -953,7 +953,8 @@ public void ModifyOptionsForImportPeptideSearchWizard(ImportPeptideSearchDlg.Wor
 
                 ProductMassAnalyzer = PrecursorMassAnalyzer;
 
-                if (workflow == ImportPeptideSearchDlg.Workflow.dia && Settings.Default.IsolationSchemeList.Count > 1)
+                if (workflow == ImportPeptideSearchDlg.Workflow.dia && Settings.Default.IsolationSchemeList.Count > 1 &&
+                    settings.TransitionSettings.FullScan.IsolationScheme == null)
                 {
                     comboIsolationScheme.SelectedIndex = 1;
```
Comment (brendanx67) [inline pwiz_tools/Skyline/SettingsUI/FullScanSettingsControl.cs:959]: Also, you should add a comment here to clarify what this is doing. i.e. What does 1 mean?

**Comment by kaipot** [comment]
> Looks good, as long as the DiaTutorialTest change is reverted.

Sorry I forgot to mention, this was a minor bug I ran into, when running the test in debug mode it caused a cross thread exception when it wasn't called from the anonymous function.
